aptos-move/framework/aptos-framework/sources/Token.move

```diff
@@ -13,27 +13,27 @@ module AptosFramework::Token {
 
     // A creator may publish multiple collections
     struct Collections<TokenType: copy + drop + store> has key {
-        collections: Table<ID, Collection<TokenType>>,
+        collections: Table<ASCII::String, Collection<TokenType>>,
```

Comment:
can we have a ID struct wrap around string instead?

Reply:
Per conversation, we went with string because it offers users a means to create meaningful uniqueness properties. The ID seems great in practice, but since it is server generated, there is no deterministic way for the user to get the ID especially since the only way to have distinct names would require us to have two tables ID -> Collection / Token and Name -> ID